codegen/echo-service/model/main.smithy

```diff
@@ -0,0 +1,54 @@
+$version: "2.0"
+
+namespace smithy.sdk.example
+
+use aws.api#service
+use aws.auth#sigv4
+use smithy.protocols#rpcv2Cbor
+ 
+@title("Echo Service")
+@service(sdkId: "Echo")
+@sigv4(name: "echoservice")
+@rpcv2Cbor
+// comment cbor and uncomment this for enabling awsJson
+// @aws.protocols#awsJson1_0
+service EchoService {
+    version: "2020-07-02"
+    operations: [EchoOperation]
+}
+
+operation EchoOperation {
+    input := with [ComplexStructureMixin] {}
+    output := with [ComplexStructureMixin] {}
```

Comment:
Just curious: why we use `ComplexStructureMixin` for both input and output

Reply:
This is an Echo service, so it returns the same shape for input and output